app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java

```diff
@@ -143,6 +145,9 @@
 
     private int leastCount, seekMax, seekMin;
     private String unit;
+    private Timer waveGenCounter;
+    private Handler powerHandler = new Handler();
```

Comment:
I meant in all places related to wave generator. This one has to be waveGenHandler or something.

Reply:
Will approve after this is changed